pkgs/development/python-modules/google-api-core/default.nix

```
@@ -19,7 +19,7 @@
 
 buildPythonPackage rec {
   pname = "google-api-core";
-  version = "2.20.0";
+  version = "2.24.0";
   pyproject = true;
 
   disabled = pythonOlder "3.6";
```

```suggestion
  disabled = pythonOlder "3.7";
```

see https://github.com/googleapis/python-api-core/blob/v2.24.0/pyproject.toml#L23